.github/workflows/main.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       uses: actions/checkout@v1
     - uses: actions/setup-go@v1
       with:
-        go-version: '1.12.10' # The Go version to download (if necessary) and use.
+        go-version: '1.13.7' # The Go version to download (if necessary) and use.
```

Comment:
Also no issues here, but bumping go version is a big enough deal that I don't want to mix it with any other changes that aren't directly related to it (like verbosity). Can you split this into another PR?

Reply:
Good point, reverted by a5170cb and 57b094f